### PR TITLE
VM-24 KQL initial

### PR DESCRIPTION
**Pull request by Monte-Hazboun**
# Overview/Summary

Ensure VMs are using Premium or Ultra disks

## Related Issues/Work Items

N/A - but I can create a work item to track in ADO if needed.

## This PR fixes/adds/changes/removes

1. KQL for VM-24 - Mission Critical Workloads should be using Premium or Ultra Disks .  Query only returns VMs that have attached disks not in the Premium or Ultra sku tier.  One line item per VM, multiple disks that meet the criteria are returned under the same VM as a list.

### Breaking Changes

none

## As part of this Pull Request I have

- [X] Read the [Contribution Guide](https://azure.github.io/Azure-Proactive-Resiliency-Library/contributing) and ensured this PR is compliant with the guide
- [X] Checked for duplicate [Pull Requests](https://github.com/Azure/Azure-Proactive-Resiliency-Library/pulls)
- [X] Associated it with relevant [GitHub Issues](https://github.com/Azure/Azure-Proactive-Resiliency-Library/issues) or ADO Work Items (Internal Only)
- [X] Ensured my code/branch is up-to-date with the latest changes in the `main` [branch](https://github.com/Azure/Azure-Proactive-Resiliency-Library/tree/main)
- [X] Ensured PR tests are passing
- [X] Performed testing and provided evidence (e.g. screenshot of output) for any changes associated to ARG queries and/or scripts
- [X] Updated relevant and associated documentation (e.g. Contribution Guide, Docs etc.)

### CODE REVIEW

**Review by ejhenry — COMMENTED** [review]
Thanks for the contribution @Monte-Hazboun! Can you please include a screenshot of the output from your query testing?

File: docs/content/services/compute/virtual-machines/code/vm-24/vm-24.kql [hunk 1]
```diff
@@ -1 +1,13 @@
-// under-development
+// Find all VMs that have an attached disk that is not in the Premium or Ultra sku tier.
```
Comment (ejhenry) [inline docs/content/services/compute/virtual-machines/code/vm-24/vm-24.kql:1]:
```suggestion
// Azure Resource Graph Query
// Find all VMs that have an attached disk that is not in the Premium or Ultra sku tier.
```

**Comment by Monte-Hazboun** [comment]
image

File: docs/content/services/compute/virtual-machines/code/vm-24/vm-24.kql (hunk 1 above)
Comment (Monte-Hazboun) [reply docs/content/services/compute/virtual-machines/code/vm-24/vm-24.kql:2]: updated

**Comment by Monte-Hazboun** [comment]
Updated picture - fixed the disk header to param1 to be consistent with the rest of the queries. 
image

File: docs/content/services/compute/virtual-machines/code/vm-24/vm-24.kql [hunk 2]
```diff
@@ -1 +1,14 @@
-// under-development
+// Azure Resource Graph Query
+// Find all VMs that have an attached disk that is not in the Premium or Ultra sku tier.
+
+Resources
```
Comment (oZakari) [inline docs/content/services/compute/virtual-machines/code/vm-24/vm-24.kql:4]:
```suggestion
resources
```

**Review by oZakari — APPROVED** [review]
LGTM 👍🏼

**Review by ejhenry — APPROVED** [review]
LGTM 👍

**Pull request merged**